scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -31,105 +31,79 @@
 import logging
 import math
 import sys
-from builtins import input
-
-import numpy
 
 from scanwaveform import dcoffset
 
 from odemis import model
 from odemis.acq.align.spot import FindGridSpots
 from odemis.util.driver import get_backend_status, BACKEND_RUNNING
+from odemis.util import transform
 
 
-def get_deflection_angle(coordinates):
+def get_scan_transform(coordinates, voltages):
     """
-    Fit a line through a set of coordinates and calculate the angle between that
-    line and the x-axis. This angle is called the deflection angle.
+    Determine the scaling transform between voltages and coordinates. The scaling transform consists of a translation,
+    rotation and scaling transformation.
 
     Parameters
     ----------
-    coordinates: (ndarray of shape nx2)
+    coordinates: (list of tuples)
         (x, y) center coordinates of the moved spot grids.
+    voltages: (list of tuples)
+        x and y offset applied to the deflectors.
 
     Returns
     -------
     phi: (float)
-        Deflection angle in radians.
+        Deflection angle in radians. Angle is counterclockwise.
```

Comment:
Can you also add where 0 is? In Odemis is 0 at 12 o'clock, which is not matching with the mathematical convention.